src/components/shared/DashboardHeader.tsx

```diff
@@ -1,18 +1,26 @@
 import React from 'react';
-import { Container, makeStyles, Typography } from '@material-ui/core';
+import {
+  Container,
+  createStyles,
+  makeStyles,
+  Theme,
+  Typography,
+} from '@material-ui/core';
 
-const useStyles = makeStyles({
-  header: {
-    'background-color': '#0D1B3B',
-  },
-  text: {
-    'font-size': '48px',
-    color: '#EEF1FB',
-    padding: '50px 0',
-    minWidth: '50%',
-    maxWidth: '1000px',
-  },
-});
+const useStyles = makeStyles(({ palette }: Theme) =>
+  createStyles({
+    header: {
+      'background-color': palette.primary.dark,
+    },
+    text: {
+      'font-size': '48px',
```

Comment:
change 'font-size' to fontSize

Reply:
Should I also change 'background-color' to backgroundColor?